editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -536,26 +533,58 @@ export const NavigatorItem: React.FunctionComponent<
     'NavigatorItem isHiddenConditionalBranch',
   )
 
+  const parentElement = useEditorState(
+    Substores.metadata,
+    (store) => {
+      const parentPath = EP.parentPath(props.navigatorEntry.elementPath)
+      return MetadataUtils.findElementByElementPath(store.editor.jsxMetadata, parentPath)
+    },
+    'NavigatorItem parentElement',
+  )
+
+  const isSlot = React.useMemo(() => {
+    const isParentConditional =
+      parentElement != null &&
+      isRight(parentElement.element) &&
+      isJSXConditionalExpression(parentElement.element.value)
+    const isNullValue =
+      isSyntheticNavigatorEntry(props.navigatorEntry) &&
+      isJSXAttributeValue(props.navigatorEntry.childOrAttribute) &&
```

Comment:
Feels a bit like this logic should be wrapped up in a function, probably using `jsxSimpleAttributeToValue` to evaluate the value itself after checking it's a `JSExpression`.

Reply:
ah yeah makes sense, I forgot this one - nice catch!